entity-management-web/src/main/java/eu/europeana/entitymanagement/web/service/ZohoSyncService.java

```diff
@@ -516,18 +564,14 @@ private void addOperation(
         // create operation if ownership is correct
         emOperation = Operations.CREATE;
       }
-    } else {
+    //even though there must be a Europeana_ID here, we check it to avoid strange zoho entries
+    } else if(Europeana_ID != null) {
       // entity record not null, update or deletion
       if (!hasDpsOwner || markedForDeletion) {
```

Comment:
this part needs to be separated, marked for deletion must be own else if statement implementing the requirements:
Check if the “Scheduled for deletion field”
If marked and no value is indicated in the “Europeana ID” field
Do nothing (ignore the entry)
If marked and the “Europeana ID” is not empty
Mark as deprecated in the API (if it isn't already) and apply an update workflow